sympy/functions/elementary/trigonometric.py

```diff
@@ -14,6 +14,7 @@
 from sympy.functions.elementary.hyperbolic import (acoth, asinh, atanh, cosh,
     coth, HyperbolicFunction, sinh, tanh)
 from sympy.sets.sets import FiniteSet
+from sympy.utilities import filldedent
```

Comment:
Some folks use it and some folks don't. I would personally use it only if the message is long (and as a side note, putting the string form of an arbitrary expression in an exception message can be an antipattern, as arbitrary expressions can be quite large and this can make raising the exception take much longer than it should). Your two exception messages here are not very long.